ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -479,8 +479,10 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
             backgroundView?.backgroundColor = .clear
         }
 
-        if !isUsingAttributedTitle {
+        if !hasOverriddenAttributedForegroundColor {
             titleView.textColor = tokenSet[.textColor].uiColor
```

Comment:
I'm not sure about this logic still--following up from @joannaquu's earlier comment, if someone applies a color to a substring, the rest will be stuck with default black color, right? Seems like the default should always be to use our Fluent color for any spot that doesn't have custom color applied.

Reply:
I'm not opposed to supporting that. Just unsure how cleanly and efficiently we can support that. I think the operation would be o(n) worst case scenario. Generally speaking, I think we could:

1. Enumerate over attributes on string to create mappings of ranges with foregroundColor attributes
2. Sort the items in the map by range
3. Starting the 0 index, find all ranges that don't have a foregroundColor attribute

Reply:
I think I found a better & easier way to do this.